project/resources/js/Components/Confirmable.vue

```diff
@@ -8,7 +8,7 @@ To use:
     </confirmable>
 -->
 <template>
-  <div>
+  <div class="inline">
```

Comment:
This will broke.
Its recommended keep without classes.

You may set the class on this utilization:
`<confirmable class="inline" ...> </coonfirmable>`